crates/goose/src/conversation.rs

```diff
@@ -0,0 +1,109 @@
+use serde::{Deserialize, Serialize};
```

Comment:
we have a message folder - we could rename that to conversation and move this there and also the conversation fixer